styles.css

```diff
@@ -1,9 +1,67 @@
-.title{
-  font-family:'Roboto', sans-serif;
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0; }
+
+.logo {
+  font:bold 24px/31px pt_serifbold, arial, helvetica, sans-serif;
+  padding: 40px 0;
+  padding-left: 44px;
+  position: relative; }
+
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  border-radius: 50%;
+  opacity: .7; }
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #c8dbfb; }
+
+.logo__circle_color_dark-blue { background: #0056ec; }
+
+.main-content__title { font-family: 'PT Serif', serif; }
+
+.wrapper {
+  margin: auto;
+  width: 80%; }
+
+.header {
+  width: 80%;
+  margin:auto;
+  display: flex;
+  justify-content: space-between; }
+
+.main-section {
+  margin: auto;
+  width: 80%;
 }
-.text{
-  font-family: 'PT+Serif', serif;
-}
-.menu{
-  font-family: 'PT+Serif', serif;
+
+.menu {
+  display: flex;
+  align-items: center; }
+
+.menu__item {
+  margin-right: 26px;
+  position: relative;
+  font:500 18px/21px Roboto, arial, helvetica, sans-serif;
+  color:#000;
+  text-decoration: none;
+ }
+
+.menu__item: last-child { margin-right: 0; }
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content:'';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6195316)